include/nbl/video/utilities/IGPUObjectFromAssetConverter.h

```diff
@@ -1648,6 +1665,8 @@ inline created_gpu_object_array<asset::ICPUDescriptorSet> IGPUObjectFromAssetCon
                     if (cpuImg)
                         cpuImg->addImageUsageFlags(asset::IImage::EUF_SAMPLED_BIT);
                     cpuImgViews.push_back(cpuImgView);
+                    if (info->info.image.sampler)
```

Comment:
more robust would be to check if this binding is not immutable in the dslayout

Reply:
> more robust would be to check if this binding is not immutable in the dslayout

Hmmm.. I partially agree.

But for this to happen, I need to rewrite this loop slightly differently. Perhaps we can also get rid of `getDescriptorInfoStorage` in the process?

Reply:
Not sure you could get rid of it, but I need the check to be performed on the layout